src/main/java/Duke.java

```diff
@@ -1,10 +1,144 @@
+    }
+
+    public static ToDo getToDo(String[] inputWords) { //returns a todo object from the user input sentence
+        String[] taskNameWords = Arrays.copyOfRange(inputWords, 1, inputWords.length);
+        String taskName = String.join(" ", taskNameWords);
+        ToDo toDo = new ToDo(taskName);
+        return toDo;
+    }
+
+    public static Deadline getDeadline(String[] inputWords) { //returns a Deadline object from the user input sentence
+        int indexOfBy = Arrays.asList(inputWords).indexOf("/by");
+        String deadlineName = String.join(" ", Arrays.copyOfRange(inputWords, 1, indexOfBy));
+        String dueDate = String.join(" ", Arrays.copyOfRange(inputWords, indexOfBy + 1, inputWords.length));
+        Deadline deadline = new Deadline(deadlineName, dueDate);
+        return deadline;
+    }
+
+    public static Event getEvent(String[] inputWords) { //returns an event object from the user input sentence
+        int indexOfAt = Arrays.asList(inputWords).indexOf("/at");
+        String eventName = String.join(" ", Arrays.copyOfRange(inputWords, 1, indexOfAt));
+        String timeOfEvent = String.join(" ", Arrays.copyOfRange(inputWords, indexOfAt + 1, inputWords.length));
+        Event event = new Event(eventName, timeOfEvent);
+        return event;
+    }
+
+    public static void printInvalidDeadline() {
+        System.out.println('\n' + "Invalid Deadline. Please provide due date!" + '\n');
+    }
+
+    public static void printInvalidEvent() {
+        System.out.println('\n' + "Invalid Event. Please provide event time!" + '\n');
+    }
+
+    public static void markTask(String[] inputWords) { //marks task as done in the list
+        int listIndex = Integer.parseInt(inputWords[1]) - 1;
+        Task markedTask = tasks.get(listIndex);
+        markedTask.setIsCompleted(true);
+        printMarkedTask(markedTask);
+    }
+
+    public static void unmarkTask(String[] inputWords){ //marks task as not done in the list
+        int listIndex = Integer.parseInt(inputWords[1]) - 1;
+        Task unmarkedTask = tasks.get(listIndex);
+        unmarkedTask.setIsCompleted(false);
+        printUnmarkedTask(unmarkedTask);
+    }
+
+    public static void addTask(String lineInput) { //adds task to list
+        Task task = new Task(lineInput);
+        tasks.add(task);
+        printAddedTask(task);
+    }
+
+    public static void addDeadline(String[] inputWords) { //adds deadline to list of tasks if keyword "/by" is found
+        if(!Arrays.asList(inputWords).contains("/by")) {
+            printInvalidDeadline();
+        }
+        else {
+            Deadline deadline = getDeadline(inputWords);
+            tasks.add(deadline);
+            printAddedTask(deadline);
+        }
+    }
+
+    public static void addEvent(String[] inputWords) { //adds event to list of tasks if keyword "/at" is found
+        if(!Arrays.asList(inputWords).contains("/at")) {
```

Comment:
This code violates coding standards, please consider adding spaces between `if` and `(`. 

As stated in the module website, the if-else class of statements should have the following form:

> if (condition) {
    statements;
}